airflow/providers/neo4j/hook.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This hook provides minimal thin wrapper around the neo4j python library to provide query execution
+"""
+import csv
+
+from neo4j import BoltStatementResult, Driver, GraphDatabase, Session
+
+from airflow.hooks.base_hook import BaseHook
+
+
+class Neo4JHook(BaseHook):
+    """
+    This class enables the neo4j operator to execute queries against a configured neo4j server.
+    It requires the configuration name as set in Airflow -> Admin -> Connections
+
+    :param n4j_conn_id: Name of connection configured in Airflow
+    :type n4j_conn_id: str
+    """
+    n4j_conn_id: str
+
+    def __init__(self, n4j_conn_id: str = 'n4j_default', *args, **kwargs):
+        self.n4j_conn_id = n4j_conn_id
+
+    @staticmethod
+    def get_config(n4j_conn_id: str) -> dict:
+        """
+        Obtain the Username + Password from the Airflow connection definition
+        Store them in _config dictionary as:
+        *credentials* -- a tuple of username/password eg. ("username", "password")
+        *host* -- String for Neo4J URI eg. "bolt://1.1.1.1:7687"
+
+        :param n4j_conn_id: Name of connection configured in Airflow
+        :type n4j_conn_id: str
+        :return: dictionary with configuration values
+        :rtype: dict
+        """
+        config: dict = {}
+        connection_object = Neo4JHook.get_connection(n4j_conn_id)
+        if connection_object.login and connection_object.host:
+            config['credentials'] = connection_object.login, connection_object.password
```

Comment:
```suggestion
            config['credentials'] = (connection_object.login, connection_object.password)
```
Just my opinion, I feel better with tuple brackets :)